# Pose2DWithSpeed.msg

float64 x # [m]
float64 y # [m]
float64 theta # [rad]
float64 speed # [m/s]
float64 lookahead_distance # [m]